docs/roadmap.md

```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
```

Comment:
Logging console is a big module, which aspect or point will be enhance within logging console? @benjaminhuo